### ISSUE TITLE
Request: Apryse

### ISSUE THREAD

**Issue by ralphangelobautista** [simple-icons/simple-icons#9656]
### Brand Name

Simple Icons

### Website

https://apryse.com/

### Popularity Metric

The Similarweb rank is: `160,836`. See: https://www.similarweb.com/website/apryse.com/

### Official Resources for Icon and Color

svg: https://docs.apryse.com/favicon.svg
color: #0206A8

**Comment by hammad-521** [comment]
Hi @uncenter ,

I came across this issue and I'm interested in contributing. Is this issue still open and in need of assistance?

Thank you!

Best regards,
Hammad

**Comment by uncenter** [comment]
Yes! Feel free to read through https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#adding-or-updating-an-icon and raise a pull request adding an icon for this. In the future if an issue like this doesn't have a linked pull request or isn't assigned to anybody you can begin work without asking.

**Comment by hammad-521** [comment]
@uncenter Could you please approve the workflows for my pull request? Thank you!
